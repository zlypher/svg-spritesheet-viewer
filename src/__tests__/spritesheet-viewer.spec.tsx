import * as React from "react";
import { SpritesheetViewer } from "../index";
import renderer from "react-test-renderer";

describe("SpritesheetViewer", () => {
    it("executes a test", () => {
        expect(1).toBe(1);
    });

    it("renders correctly and matches the snapshot", () => {
        const component = renderer.create(
            <SpritesheetViewer />
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
