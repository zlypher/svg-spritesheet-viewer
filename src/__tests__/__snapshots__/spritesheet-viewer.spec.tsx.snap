// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpritesheetViewer renders correctly and matches the snapshot 1`] = `
<section>
  <div>
    <svg>
      <use
        xlinkHref="#test-one"
      />
    </svg>
    <span>
      test-one
    </span>
  </div>
</section>
`;
